src/main/java/minicraft/core/Renderer.java

```diff
@@ -236,17 +229,18 @@ private static void renderLevel() {
 	private static void renderGui() {
 		// This draws the black square where the selected item would be if you were holding it
 		if (!isMode("minicraft.settings.mode.creative") || player.activeItem != null) {
-			for (int x = 10; x < 26; x++) {
-				screen.render(x * 8, Screen.h - 8, 5, 2, 0, hudSheet.getSheet());
+			for (int y = 0; y < 2; y ++) {
+				for (int x = 0; x <= 280; x++) {
+					screen.render(x, Screen.h - 16 + y * 8, 5, 2, 0, hudSheet.getSheet());
+				}
 			}
 		}
 
 		// Shows active item sprite and name in bottom toolbar.
 		if (player.activeItem != null) {
-			player.activeItem.renderHUD(screen, 10 * 8, Screen.h - 8, Color.WHITE);
+			player.activeItem.renderHUD(screen, 14 * 8, Screen.h - 8, Color.WHITE);
```

Comment:
What's the point of these changes?

Reply:
It brings back the beautiful, full-width and two rows height black rectangle for the status icons. I think that the new bar (that my change overwrites) looks a bit bald and icons without any background looks crappy. It would be so much better to remove the black rectangle at all, however without the rectangle the selected item status looks kind of bad design, and doesn't even fit in minicraft's theme. That is why I decided to change dimensions of the status rectangle. I would really appreciate it, if you let the new code in, or at least make a survey for the community, so they could decide which one fits better.

Reply:
The pull request won't be accepted if it contains changes that are outside the scope of the pull request.

Reply:
what about the first, [143d5e7](https://github.com/MinicraftPlus/minicraft-plus-revived/pull/474/commits/143d5e77b1c773ccff9fd8fce2ce2ca9d8e1a7d9) commit?

Reply:
First and second commits are okay.

Reply:
Then why the first commit is not applied?

Reply:
That is not how pull requests work. It's all or none.

Reply:
I think I have commited a change that overwrites the recent status bar commit